src/cargo/ops/cargo_package.rs

```diff
@@ -26,6 +26,9 @@ pub struct PackageOpts<'cfg> {
     pub verify: bool,
     pub jobs: Option<u32>,
     pub target: Option<String>,
+    pub registry: Option<String>,
```

Comment:
Why is registry added if it is never used?

Reply:
Good catch!

I copied the registry field over from somewhere else and then implemented the init without realising it was actually never used :sweat_smile: